packages/client/src/App.js

```diff
@@ -12,30 +12,30 @@ import { ProductPage } from './containers/ProductPage/ProductPage.Container';
 import { ContactUsFeedbackPage } from './containers/ContactUsPage/ContactUsFeedbackPage.Container';
 import Navigation from './components/Navigation/Navigation.component';
 import { Footer } from './components/Footer/Footer.component';
+import { SearchedProducts } from './containers/SearchedProducts/SearchedProducts';
 
 function App() {
   return (
-    <CartStateProvider>
```

Comment:
If you remove this, everything related to the shopping cart breaks.

Reply:
Oops, accidentally it's changed, thanks